frontend/src/admin/common/tables/SortedTable.js

```diff
@@ -1,6 +1,16 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
+
 import Table from './Table';
-import sortColumn from '../../lib/sortColumn';
+import sortColumn from '../../../lib/sortColumn';
```

Comment:
I know it was kinda already like this, but the deeply relative import jumps out at me. As does the name 'lib', which I suspect is thinly veiled 'util' module. 😝

I wonder if `sortColumn` could come into the `common/tables` folder? It was probably only pulled out to 'lib' because we used to have lots of different table components.